auth-api/src/auth_api/models/user.py

```diff
@@ -70,30 +73,38 @@ def create_from_jwt_token(cls, token: dict):
             current_app.logger.debug(
                 'Creating user from JWT:{}; User:{}'.format(token, user)
             )
-            db.session.add(user)
-            db.session.commit()
+            user.save()
             return user
         return None
 
     @classmethod
-    def find_by_username(cls, username):
-        """Return the first User for the provided username."""
-        return cls.query.filter_by(username=username).first()
+    def update_from_jwt_token(cls, token: dict):
+        """Update a User from the provided JWT."""
+        if token:
+            user = User.find_by_jwt_token(token)
+            if user:
+                user.username = token.get('preferred_username', user.username)
+                user.firstname = token.get('firstname', user.firstname)
+                user.lastname = token.get('lastname', user.lastname)
+                user.email = token.get('email', user.email)
+                user.modified = datetime.datetime.now()
```

Comment:
is there a way in python to set modified to be auto update on every updates?something which wil go in the model and we dont have to write it in every update method

Reply:
Probably, but I don't know what that is.  Would be a better way to do it though, so I will look into it.

Reply:
`@validates` provides option to validate any field before persisting.

Reply:
Thanks @pksumesh ! Will check that out

Reply:
This change will require another database migration, so going to include it in the work being done for Org API.